docs/content/src/learn/tutorials/saas-integrations/amazonsqs/notifying-fire-alarm-using-sqs/README.md

```diff
@@ -52,40 +52,53 @@ To implement the scenario in this guide, you can use the following package struc
 
 Now that you have created the project structure, the next step is to develop the integration scenario.
 
-#### Write the integration.
+3. Set up credentials for Amazon SQS
```

Comment:
The formatting looks messy when we render this. Shall we follow the heading approach used in other tutorials?